packages/synapse-interface/pages/returntomonke/ImageUploader.tsx

```diff
@@ -0,0 +1,134 @@
+import _ from 'lodash'
+import { useState, useRef, useEffect } from 'react'
+import Button from '@tw/Button'
+
+export default function ImageUploader() {
+  const [uploadedImage, setUploadedImage] = useState(null);
+  const [processedImage, setProcessedImage] = useState(null);
+
+  const fileInput = useRef(null);
+  const imgRef = useRef(null);
+
+  useEffect(() => {
+    if (uploadedImage) {
+      applyImageOverlay();
+    }
+  }, [uploadedImage]);
+
+  function handleFile(file) {
+    setUploadedImage(file);
+  }
+
+  function handleDragOver(event) {
+    event.preventDefault();
+  }
+
+  function handleOnDrop(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    handleFile(event.dataTransfer.files[0]);
+    console.log('File drop was a success');
+  }
+
+  const applyImageOverlay = () => {
+    const canvas = imgRef.current;
+    const ctx = canvas.getContext('2d');
+
+    // Clear the canvas
+    ctx.clearRect(0, 0, canvas.width, canvas.height);
+
+    // Load the uploaded image
+    const img = new Image();
+    img.src = URL.createObjectURL(uploadedImage);
+    img.onload = () => {
+      // Set the canvas size to match the uploaded image
+      canvas.width = img.width;
```

Comment:
Currently uploading an image that doesn't have equal image width and height will cause for the PFP to distort (see screenshots attached below) 

Can test out here: https://fe-center-synape-pf.sanguine-fe.pages.dev/returntomonke

To fix this to always create a square PFP output, we could set the `canvas.width` and `canvas.height` equal to each other. 

As an example, updated the `applyImageOverlay()` function to achieve that effect: https://github.com/synapsecns/sanguine/pull/2465/files#diff-1f5949bc4b332bdd1d21bc1fccf989474ccf9b40c36d9b78bbae84a5f7a89b71R44-R50

Given that an uploaded image could differ in height or width, chose to set both `canvas.width` and `canvas.height` to whichever was greater, `image.width` or `image.height` so that we could always generate a square PFP output image. 

![synape_Screenshot 2024-04-04 a
<img width="78" alt="Screenshot 2024-04-04 at 1 07 05 PM" src="https://github.com/synapsecns/sanguine/assets/57741810/5c70672f-4375-4bba-8cbf-358695540fef">
t 1](https://github.com/synapsecns/sanguine/assets/57741810/39c0ed0e-aab1-45aa-8798-141711344dcd)